### DISCUSSION TITLE
Image Viewer

### DISCUSSION THREAD

**Discussion by ArsMasiuk** [doublecmd/doublecmd#1385] [Suggestions]
It would be nice to add some features to the Image Viewer:
- select white or dark color of background (because there could be icons which are mostly white so they are hard to see on the white background)
- extend keyboard navigation by PgUp/PgDown/Home/End etc.
- double-click on the preview could open an external editor
- single-click on the preview could follow the image file in the active commander's panel
- zoom with control+wheel

**Comment by Skif-off** [comment]
It would be more convenient with numbers (without unnecessary quoting)...

> select white or dark color of background

[Enable transparency](https://doublecmd.github.io/doc/en/viewer.html#mnu_image). Also go to "Configuration" in the main menu > Options... > Colors > the Viewer category.

> double-click on the preview could open an external editor

Maybe the ability to specify several applications? + maybe menu item that opens a file according to file associations (system or [internal](https://doublecmd.github.io/doc/en/configuration.html#ConfigAssociations)/system)?
(Toolbar will be available in the Full Screen mode too.)

> single-click on the preview could follow the image file in the active commander's panel

Then how to select a file to view?

> zoom with control+wheel

Doesn't this work?

P.S. I mean DC 1.1.0+.
